plugin/storage/grpc/config/config.go

```diff
@@ -15,42 +15,31 @@
 package config
 
 import (
-	"context"
-	"fmt"
-	"os/exec"
 	"time"
 
-	"github.com/hashicorp/go-hclog"
-	"github.com/hashicorp/go-plugin"
-	"go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"
 	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials"
-	"google.golang.org/grpc/credentials/insecure"
 
-	"github.com/jaegertracing/jaeger/pkg/config/tlscfg"
-	"github.com/jaegertracing/jaeger/pkg/tenancy"
 	"github.com/jaegertracing/jaeger/plugin/storage/grpc/shared"
 )
 
 var pluginHealthCheckInterval = time.Second * 60
 
 // Configuration describes the options to customize the storage behavior.
-type Configuration struct {
-	PluginBinary            string `yaml:"binary" mapstructure:"binary"`
-	PluginConfigurationFile string `yaml:"configuration-file" mapstructure:"configuration_file"`
-	PluginLogLevel          string `yaml:"log-level" mapstructure:"log_level"`
-	RemoteServerAddr        string `yaml:"server" mapstructure:"server"`
-	RemoteTLS               tlscfg.Options
-	RemoteConnectTimeout    time.Duration `yaml:"connection-timeout" mapstructure:"connection-timeout"`
-	TenancyOpts             tenancy.Options
-
-	pluginHealthCheck     *time.Ticker
-	pluginHealthCheckDone chan bool
-	pluginRPCClient       plugin.ClientProtocol
-	remoteConn            *grpc.ClientConn
-}
+// type Configuration struct {
+// 	PluginBinary            string `yaml:"binary" mapstructure:"binary"`
+// 	PluginConfigurationFile string `yaml:"configuration-file" mapstructure:"configuration_file"`
+// 	PluginLogLevel          string `yaml:"log-level" mapstructure:"log_level"`
+// 	RemoteServerAddr        string `yaml:"server" mapstructure:"server"`
+// 	RemoteTLS               tlscfg.Options
+// 	RemoteConnectTimeout    time.Duration `yaml:"connection-timeout" mapstructure:"connection-timeout"`
+// 	TenancyOpts             tenancy.Options
```

Comment:
These are not plugin options, they are for remote grpc storage that we will continue to support